src/resources/views/store/s3.blade.php

```diff
@@ -24,10 +26,10 @@
         @error('endpoint')
             <p class="help-block">{{$message}}</p>
         @enderror
-        <p class="help-block">
-            This must be the full URL including the bucket name, region etc.
-        </p>
     </div>
+    <p class="help-block">
+        This must be the full URL including the bucket name, region etc.
+    </p>
```

Comment:
See above.

Reply:
@mzur 
I moved it because previously, if an error existed, the text "This must be..." would also be highlighted in red.